htlcswitch/link.go

```diff
@@ -1736,6 +1737,11 @@ func (l *channelLink) Peer() lnpeer.Peer {
 	return l.cfg.Peer
 }
 
+// ChannelPoint returns the channel outpoint for the channel link.
```

Comment:
add ` // NOTE: Part of the ChannelLink interface.`

Reply:
fixed